### PR TITLE
[EthFlow]#1294 - updated backend return types

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Part of #1294 

Wiring up refund tx hash

After the order expires and the automated refunder kicks in, the refund tx hash will be displayed at the bottom of the 3rd step
Screenshot 2022-12-26 at 16 25 11

  # To Test

Follow same test steps as https://github.com/cowprotocol/cowswap/pull/1756

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
Hey @alfetopito , great job!

Just a nitpick: maybe we should not duplicate 'ETH refunded' for the link name, and just show 'View transaction' instead, WDYT?
image

**Comment by alfetopito** [comment]
> Hey @alfetopito , great job!
> 
> Just a nitpick: maybe we should not duplicate 'ETH refunded' for the link name, and just show 'View transaction' instead, WDYT? image

Great idea

I've update it to match your suggestion:

Screenshot 2022-12-27 at 10 06 14

**Comment by elena-zh** [comment]
LGTM!

File: src/cow-react/pages/Faq/EthFlowFaq.tsx [hunk 1]
```diff
@@ -289,8 +289,8 @@ export default function EthFlowFAQ() {
               How can I check I actually received the refund after my order expired?
             </h3>
             <p>
-              You can click on the hyperlink “ETH refunded successfully” on the progress modal which will redirect you
-              to etherscan.io to see the actual transaction on the blockchain.
+              You can click on the hyperlink “View transaction” on the progress modal which will redirect you to
```
Comment (anxolin) [inline src/cow-react/pages/Faq/EthFlowFaq.tsx:292]: Great you remembered 🤩

**Review by anxolin — APPROVED** [review]

**Pull request merged**